### DISCUSSION TITLE
NPM package landing pages not respecting repository.directory in package.json - is there another way?

### DISCUSSION THREAD

**Discussion by JamesHenry** [npm/rfcs#215] [Question]
Reported here: https://github.com/typescript-eslint/typescript-eslint/issues/879

For typescript-eslint we have a monorepo, and the links on a README.md file are within that context.

Despite us specifying the repository.directory within the relevant package.json, the NPM landing page for the package does not use then when building the link, so we have the case where the link on github works, but not the one on NPM.

If this is acknowledged to be a "bug" in the site, then please advise on where to report it?

Many thanks!

**Comment by robsonos** [comment]
I am having the same issue. My first release had a different `repository.directory ` than the latest release, and now all references in docs point to the old one. Is there any workaround for this?
